docs/configuration.rst

```diff
@@ -0,0 +1,108 @@
++---------------------------+--------------------------------------------------+
+|   UPLOADED_FILES_DENY     | This lets you deny file extensions allowed by    | 
+|                           | the upload set in the code.                      |
++---------------------------+--------------------------------------------------+
+
+
+To save on configuration time, there are two settings you can provide
+that apply as "defaults" if you don't provide the proper settings otherwise.
+
+
++---------------------------+--------------------------------------------------+
+|         Config Key        |                 Description                      |
++===========================+==================================================+
+|    UPLOADS_DEFAULT_DEST   | if an upload set's destination isn't declared,   |
+|                           | then its uploads will be stored in               |
+|                           | a subdirectory of this directory.                |
+|                           | For example, if you set this to ``/var/uploads``,|
+|                           | then a set named photos will store its uploads   |
+|                           | in ``/var/uploads/photos``.                      |
++---------------------------+--------------------------------------------------+
+|   UPLOADS_DEFAULT_URL     | If you have a server set up to serve from        |
+|                           | `UPLOADS_DEFAULT_DEST`, then set the server's    | 
+|                           | base URL here. Continuing the example above, if  |
+|                           | ``/var/uploads`` is accessible from              |
+|                           | ``http://localhost:5001``, then you would set    |
+|                           | this to ``http://localhost:5001/`` and URLs for  |
+|                           | the photos set would start with                  |
+|                           | ``http://localhost:5001/photos``.                |
+|                           | Include the trailing slash.                      |
++---------------------------+--------------------------------------------------+
+
+You should ensure that each `UploadSet` you are creating had set its distenation
+directory, unless you will get `RuntimeError` exception. To avoid this error you
+should set `UPLOADED_[SETNAME]_DEST` or `UPLOADS_DEFAULT_DEST`. 
+
+
+Autoserve configuration:
+------------------------
+
+`UPLOADS_AUTOSERVE`
+Setting this configuration to `True` enables automatic viewing/downloading of uploaded files.
+When the name of the `UploadSet` is `photos` and the name of the uploaded file
+is `snow.jpg`, the file is available via:
+
+    .. code-block:: python
+
+        flask.url_for('_uploads.uploaded_file', setname='photos', filename='snow.jpg')
+    
+
+which resolve to:
+    
+    `http://localhost:5000/_uploads/photos/snow.jpg`
+
+
+``Default``: In order to stay compatible with `Flask-Uploads`, for
```

Comment:
I'm not sure about the appearance if I accept this suggestion.
I want to go with something like: 

> Default Value: In order to stay compatible with `Flask-Uploads`, for

Reply:
You can always do a `git pull` and then locally do a `tox -e docs` to render the documentation on your pc/laptop. I think it might even be possible to generate a preview here in github, but I do not know how to do that.

Reply:
``Default Value: In order to stay compatible with `Flask-Uploads`, for 
`Flask-Reuploaded<1.0.0` the `UPLOADS_AUTOSERVE` default is `True`. 
Since version `1.0.0` it is `False` by default.``

This is the last result that gives acceptable appearance in my opinion.
Thanks for help.